relayer/src/supervisor/spawn.rs

```diff
@@ -407,7 +407,7 @@ impl<'a, Chain: ChainHandle + 'static> SpawnContext<'a, Chain> {
         );
 
         if conn_state_src.is_open() && conn_state_dst.is_open() {
-            debug!(
+            trace!(
                 "connection {} on chain {} is already open, not spawning Connection worker",
                 connection.connection_id,
                 chain.id()
```

Comment:
Both args here could usefully be KVs:

```suggestion
            trace!(
                connection_id=%connection.connection_id,
                chain_id=%chain.id(),
                "connection is already open, not spawning Connection worker"
```